src/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -214,7 +214,7 @@ public unsafe bool IsFocused
             }
         }
 
-        private bool _isVisible;
+        private readonly bool _isVisible;
```

Comment:
This shouldn't be read-only either, it's a bug that this never gets set. Similar to: #1076

Reply:
readonly keyword removed